### ISSUE TITLE
cmake issue

### ISSUE THREAD

**Issue by FesterandTux** [i-rinat/libvdpau-va-gl#49]
I am following your vdpau instructions and getting a cmake error:
The source directory "/home/joesmith/build" does not appear to contain CMakeLists.txt

**Comment by FesterandTux** [comment]
forget it.  I got past that.  (Never used cmake before).

**Comment by FesterandTux** [comment]
Actually I have no idea if your package will help me.  compiling this is beyond my skillset.  
This is what I'm looking for.  I am running mythtv (mythbuntu) using the VDPAU video.  I am upgrading from 12.04 to 14.04.  In 12.04 I was using a gt430 video card, but in 14.04 I want to remove the gt430 and use my amd a10-6800k APU.  Yet I believe I need VDPAU for my videos to look right in mythtv.  After much searching, I found your solution but I can't install it.

**Comment by Yukariin** [comment]
@FesterandTux why you not use [binary package](http://packages.ubuntu.com/trusty/libvdpau-va-gl1) from ubuntu repository?

**Comment by i-rinat** [comment]
For some years now open-source AMD drivers have native VDPAU support, so you don't need `libvdpau-va-gl` for that.

If you are using proprietary AMD drivers, you should have either native XvBA support (VA-API adapter exists), or native VA-API support (for more than a year now). In that case `libvdpau-va-gl` can be used to get VDPAU interface. But since MythTV [have VA-API support](https://www.mythtv.org/wiki/VAAPI), it's better to use that than stack API converters one onto another.

Long story short, for MythTV, there are easier ways exists than using this project.

**Comment by FesterandTux** [comment]
Thank you both for your help.  I ended up reinstalling mythbuntu 14.04 and using the generic Radeon drivers.  Working much better now.  Thanks!
